vendor/engines/secure_rooms/spec/features/scans_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe "Scanning in and out", type: :controller do
```

Comment:
I wasn't quite sure what to do with this. My goal was to have a full-stack happy path, but it's hard to do API requests in feature specs. And this seemed somewhat different than the other ScansController controller specs. I'm open to other ideas.